sdk/lib/crt/math/arm/__dtoi64.s

```diff
@@ -1,23 +1,33 @@
 /*
- * COPYRIGHT:         BSD - See COPYING.ARM in the top level directory
- * PROJECT:           ReactOS CRT library
- * PURPOSE:           Implementation of __dtoi64
- * PROGRAMMER:        Timo Kreuzer (timo.kreuzer@reactos.org)
+ * PROJECT:     ReactOS CRT library
+ * LICENSE:     MIT (https://spdx.org/licenses/MIT)
+ * PURPOSE:     Implementation of __dtoi64
+ * COPYRIGHT:   Copyright 2015 Timo Kreuzer <timo.kreuzer@reactos.org>
+ *              Copyright 2021 Roman Masanin <36927roma@gmail.com>
  */
 
 /* INCLUDES ******************************************************************/
 
 #include <kxarm.h>
 
+    IMPORT __dtoi64_worker
+
 /* CODE **********************************************************************/
 
     TEXTAREA
 
     LEAF_ENTRY __dtoi64
+    /* Allocate stack space and store parameters there */
+    push {lr}
+    PROLOG_END
```

Comment:
I think it's worth to add comments into all assembly files which parameters are in and out, just like Timo did here: https://github.com/reactos/reactos/pull/3841/files#diff-c7acfed7ac82fae423f142361a963cf55283623e879cc0ea70bab7e3b0589288R19-R24